src/checkout-buttons/create-checkout-button-registry.spec.ts

```diff
@@ -12,9 +12,10 @@ import { GooglePayButtonStrategy } from './strategies/googlepay';
 
 describe('createCheckoutButtonRegistry', () => {
     let registry: Registry<CheckoutButtonStrategy>;
+    const locale = 'en';
 
     beforeEach(() => {
-        registry = createCheckoutButtonRegistry(createCheckoutStore(), createRequestSender(), createFormPoster());
+        registry = createCheckoutButtonRegistry(createCheckoutStore(), createRequestSender(), createFormPoster(), locale);
```

Comment:
Same here, instead of assigning to a temporary variable, you could directly pass it to the function